app/src/main/java/com/example/atheneum/models/Notification.java

```diff
@@ -207,4 +210,31 @@ public void constructMessage(String userName, String bookName) {
             this.message = userName + " has declined your request for the book " + bookName;
         }
     }
+
+    /**
+     * Returns isSeen boolean
+     *
+     * @return boolean indicating whether user has clicked this notification
+     */
+    public boolean getIsSeen() {

```

Comment:
Since you added new methods to Notification, you should update the unit tests as well so we don't get docked marks.

Reply:
Updated in recent commit